modules/nf-scil/tracking/localtracking/meta.yml

```
@@ -0,0 +1,67 @@
+---
+name: "tracking_localtracking"
+description: process local tracking
+keywords:
+  - Tractography
+  - Local tracking
+tools:
+  - "Scilpy":
+      description: "The Sherbrooke Connectivity Imaging Lab (SCIL) Python dMRI processing toolbox."
+      homepage: "https://github.com/scilus/scilpy.git"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+  
+  - fodf:
+      type: file
+      description: Nifti image of spherical harmonic file (fodf)
+      pattern: "*.{nii,nii.gz}"
+
+  - fa:
+      type: file
+      description: NIfti image binary mask for fa mask
+      pattern: "*.{nii,nii.gz}"
+
+  - wm:
+      type: file
+      description: Nifti image white matter mask.
+      pattern: "*.{nii,nii.gz}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+  
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+  - trk:
+      type: file
+      description: Tractogram output file.
+      pattern: "*.{trk,tck}"
+
+  - seeding:
```

Match this with the name of your output in your main.nf
```suggestion
  - seedmask:
```

---

done